package.json

```diff
@@ -8,7 +8,7 @@
     "fix": "prettier --write .",
     "test": "jest",
     "start": "node server.js",
-    "dev": "local-ssl-proxy --source 443 --target 5500 -n dev.realdevsquad.com"
+    "dev": "node server.js & local-ssl-proxy --source 443 --target 8000 -n dev.realdevsquad.com"
```

Comment:
- couple of questions:
   - what does `node server.js` do?
   - why are we changing the port from 5500 to 8000?

Reply:
i'm not sure i understand why all this question. i basically made a change to combine 2 script instead of have user run 1 and get lost (mostly new user).